doc/README.logstash

```diff
@@ -13,6 +13,73 @@ $ ntopng -F "logstash;localhost;tcp;5510"
 Syntax:
 logstash;<host>;<protocol>;<port_number>
 
+ntopng will NOT create the mappings automatically - a template should be created. 
```

Comment:
Why you keep putting this Elastic Search template configuration? This is not necessary in logstash. Please remove it.

Also note that, with ES export enabled, this index template is automatically pushed to ES.